<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblDayOfWeek">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Day of week</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QTableView" name="tblSerie1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>133</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlMovements">
     <item>
      <widget class="QLabel" name="lblPushups">
       <property name="text">
        <string>Pushups</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSquats">
       <property name="text">
        <string>Squats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPullups">
       <property name="text">
        <string>Pullups</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblLegRaises">
       <property name="text">
        <string>LegRaises</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBridges">
       <property name="text">
        <string>Bridges</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblHandstandPushups">
       <property name="text">
        <string>HandstandPushups</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>91</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlRepetitions">
     <item>
      <widget class="QSpinBox" name="spnPushups">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnSquats">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnPullups">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnLegRaises">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnBridges">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnHandstandPushups">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1011</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlUser">
     <item>
      <widget class="QComboBox" name="cmbUser"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDayOfWeek"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start Routine</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlRoutine">
     <item>
      <widget class="QRadioButton" name="rbtnNewBlood">
       <property name="text">
        <string>new blood</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtnGoodBhavior">
       <property name="text">
        <string>good behavior</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtnVeteran">
       <property name="text">
        <string>veteran</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblRoutine1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblRoutine2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tblSerie2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tblSerie3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tblSerie4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tblSerie5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>620</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tblSerie6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>690</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdRest">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbRoutine1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbRoutine2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>248</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>229</height>
      </rect>
     </property>
     <widget class="QFrame" name="vfImage1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vlImage1">
       <item>
        <widget class="QLabel" name="lblImage1">
         <property name="text">
          <string>Image1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>229</height>
      </rect>
     </property>
     <widget class="QFrame" name="vfImage2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblImage2">
         <property name="text">
          <string>Image2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
